lib/msal-browser/docs/logging.md

```diff
@@ -0,0 +1,52 @@
+# MSAL JS logging instructions
+
+These are steps needed to enable MSAL JS logging for your applications:
+
+1. In the MSAL [Configuration](./configuration.md) object, you can enable logging to collect msal js logs. We enable different levels of logging and an appropriate level can be chosen as needed. 
+
+2. The logger options can be set as below, the example chooses `LogLevel.Trace`
+
+```javascript
+
+    ...
```

Comment:
nit: For completeness we should show that this is the object that gets passed into `PublicClientApplication`

Reply:
+1

Reply:
Done